backend/test/db.test.ts

```diff
@@ -145,14 +140,16 @@ describe('db init', () => {
     it('should handle array of URLs', () => {
         const sequelizeMock = mockSequelize();
         const url = 'postgres://url';
-        const initializer = new DbInitializer(getOptions([url], { ca: 'caPath' }), mockLogger(), modelFactories);
-        request.mockImplementationOnce((options, handler) => handler(true));
-        request.mockImplementationOnce((options, handler) => handler(false, { statusCode: 200 }));
-        return initializer.init().then(() => {
+        const dbModule = getDbModuleWithMockedLogger(getOptions([url], { ca: 'caPath' }), mockLogger());
+        (<jest.Mock>(<unknown>request)).mockImplementationOnce((_options, handler) => handler(true));
+        (<jest.Mock>(<unknown>request)).mockImplementationOnce((_options, handler) =>
+            handler(false, { statusCode: 200 })
+        );
+        return dbModule.init().then(() => {
             expect(request).toHaveBeenCalledTimes(2);
-            expect(request.mock.calls[0][0]).toMatchObject({ url: 'https://url:8008' });
-            expect(request.mock.calls[1][0]).toMatchObject({ url: 'https://url:8008' });
-            expect(Sequelize.mock.calls[0][0]).toBe(url);
+            expect((<jest.Mock>(<unknown>request)).mock.calls[0][0]).toMatchObject({ url: 'https://url:8008' });
+            expect((<jest.Mock>(<unknown>request)).mock.calls[1][0]).toMatchObject({ url: 'https://url:8008' });
+            expect((<jest.Mock>(<unknown>Sequelize)).mock.calls[0][0]).toBe(url);
```

Comment:
I'm not sure about that one. I think the intention of the creator was to check only single arguments and not all arguments. With `toHaveBeenCalled` it's not possible, so I would need to add artificial checks like `expect.any(Function)`. 

I think it would be better to find a solution for not having to cast to `unknown` instead of changing the verification method. 
I don't think I would have time to dig deeper right now.

Reply:
I don't see any issue with using `expect.any` or even `expect.anything`.

Alternatively, maybe we could have just a simple function that would perform the casting? Something like `toMock`. Or even better for this particular case `getMockCalls` that would return `mock.calls` for the given argument.

Reply:
Added `toMock` function to perform casting.